### ISSUE TITLE
Could not find prepared statement with handle 0

### ISSUE THREAD

**Issue by NickNeck** [livehelpnow/tds#45]
Hello, 
`Tds.query` raises the exception "Could not find prepared statement with handle 0". 
I have added a test for this in PR #44.

It seems that the exception will be raised if `sp_prepare` returns a `@handle` with value `0`.

**Comment by mjaric** [comment]
Thanks, I run once on this issue but haven't figured out what was the case. I will look on it.

**Comment by woylie** [comment]
The `sp_prepare` handles have the type `int` (4 bytes), yet the problem seems to occur whenever a handle is greater than 255. Is it possible that the handle parameter is encoded or decoded as a `tinyint` (1 byte) somewhere?

**Comment by woylie** [comment]
And by the way, I've noticed the `int_type_size` function in `types.ex`:

```elixir
defp int_type_size(int) when int == nil, do: 4
defp int_type_size(int) when int in 0..255, do: 4
defp int_type_size(int) when int in -32768..32767, do: 2
defp int_type_size(int) when int in -2147483648..2147483647, do: 4
defp int_type_size(int) when int in -9223372036854775808..9223372036854775807, do: 8
```

Is there a particular reason why the size for values between 0 and 255 is set to 4, and not to 1?

**Comment by mjaric** [comment]
released with v1.0.8.
